Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp

```diff
@@ -225,8 +225,6 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian(
                         // check the normalized error
                         if (amrex::Math::abs(1._rt - M_magnitude_normalized) > mag_normalized_error)
                         {
-                            printf("i = %d, j=%d, k=%d\n", i, j, k);
-                            printf("M_magnitude_normalized = %f, mag_normalized_error=%f\n", M_magnitude_normalized, mag_normalized_error);
```

Comment:
removing print statements here as SYCL did not compile and threw an error

Reply:
I saw that; thank you!